User Guide/Introduction.md

```diff
@@ -16,16 +16,25 @@ There are several roles within the ADEX application.
   - **Browse all topics** published by publisher from **other organisations** based on the visibility **"Only my team"**
   - **Publish** and **Subscribe** to topics
   - **Manage teams** such as Create new teams, Edit team details, Add team members, Edit role of team members and Leave team
+  - **Join team** in any organisation, and first time user role will be **Viewer** for the team.
   
 - **Publisher**	
 
   - Browse topics and verify topic details
   - Create new team, Create projects and topics
+  - Join a team in any org as **Viewer**
```

Comment:
"org" -> "organisation"

Reply:
Create new team, Join a team in any organisation.

Reply:
ok KC